docs/custom_metrics_hpa.md

```diff
@@ -0,0 +1,39 @@
+### Enabling custom metrics hpa for starboard exporter
+
+#### Required components
+
+Prometheus adapter
+
+#### Steps
+
+1. Install prometheus adapter.
+    1. Configure helm chart to connect with prometheus to get the metrics
+        ```
+        prometheus:
+          # Value is templated
+          url: http://prometheus-operated.gaia-prometheus.svc
+          port: 9090
+          path: "/gaia"
+        ```
+    2. Create seriesQuery & metricsQuery accordingly to fetch metrics of the required component. For example,
+        ```
+         rules:
+           custom:
+             - seriesQuery: 'scrape_duration_seconds{app="starboard-exporter"}'
+               seriesFilters: []
+               resources:
+                 template: <<.Resource>>
+               name:
+                 as: "scrapedurationseconds"
+               metricsQuery: scrape_duration_seconds{app="starboard-exporter"}
```

Comment:
`{app="X"}` doesn't work outside our MCs, right?

Reply:
Yes, I updated it in the docs & added short description too